### PR TITLE
added db connection string support

### LINKED ISSUE #122: Add MongoDb connection details to settings
We need additional values to connect to mongo cosmos

### PR DESCRIPTION
**Pull request by SteveMaier-IRT**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
Fixes #122

### Description
Added display and setting for the db connection string.

### CODE REVIEW

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/SettingsViewModel.cs [hunk 1]
```diff
@@ -14,13 +16,25 @@ public SettingsViewModel(IServiceProvider serviceProvider) : base(null, serviceP
...
     {
         // save db settings and reset the db connection
         Preferences.Set("DbAddress", DatabaseAddress);
         Preferences.Set("DbPassword", DatabasePassword);
-        DbService.Init(DatabaseAddress, DatabasePassword);
+        Preferences.Set("DbConnection", DatabaseConnectionString);
+
+        if (!string.IsNullOrEmpty(DatabaseConnectionString))
+        {
+            DbService.Init(DatabaseConnectionString);
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/SettingsViewModel.cs:32]: To reduce user confusion, if we save with a connection string, can we blank out Address / Pw?

File: src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs [hunk 2]
```diff
@@ -42,6 +42,7 @@ public static class DbService
 
     private static string DbHost = string.Empty;
     private static string DbPassword = string.Empty;
+    private static string DbConnection = string.Empty;
```
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs:45]: These should be moved into `Init(host, password)` as constants / parameters.

**Review by john-s-morgan — CHANGES_REQUESTED** [review]
Scope cleanup for Init

File: src/electionguard-ui/ElectionGuard.UI/ViewModels/SettingsViewModel.cs (hunk 1 above)
Comment (john-s-morgan) [inline src/electionguard-ui/ElectionGuard.UI/ViewModels/SettingsViewModel.cs:32]: To reduce user confusion, can we clear username / password if a connection string is present?

File: src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs (hunk 2 above)
Comment (john-s-morgan) [reply src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs:45]: per convo, this will be left as private statics for future reconnect logic

Comment (SteveMaier-IRT) [reply src/electionguard-ui/ElectionGuard.UI.Lib/Services/DbService.cs:45]: reconnect method added

**Pull request merged**